app/src/main/java/com/bluedotinnovation/realtimesyncsampleapp/fragments/LogFragment.java

```diff
@@ -155,13 +154,34 @@ public void run() {
 
     }
 
-
     @Override
     public void onPause() {
         super.onPause();
         MainActivity.LOG_DATA = tvLog.getText().toString();
     }
 
+    @Override
+    public void onInitializationFinished(@org.jetbrains.annotations.Nullable BDError bdError) {
+        if (bdError == null) {
+            updateLog("Bluedot Point SDK authenticated");
+            GeoTriggeringService.builder().start(getContext(), this);
```

Comment:
Integration example should have Foreground Notification implementation of GeoTrigger